rps.js

```diff
@@ -0,0 +1,17 @@
+const rpsList = ['rock', 'paper', 'scissors'];
+const user = 'rock';
+const rps = function (a) {
+    const b = rpsList[Math.floor(Math.random() * 3)];
+    console.log("computer selected " + b);
+    if (a === b)
+        return "tie"
+    if ((a == 'rock' && b == 'scissors') || (a == 'scissors' && b == 'paper') || (a == "paper" && b == "rock")) {
+        return "win";
+    }
+    if ((b == 'rock' && a == 'scissors') || (b == 'scissors' && a == 'paper') || (b == "paper" && a == "rock")) {
```

Comment:
You can make the code even shorter by changing this if condition with with ```else { return false}```, since at this point all the possible outcome left is the losing ones so you don't need to check for them